pyartcd/pyartcd/constants.py

```diff
@@ -17,8 +17,15 @@
     "ppc64le": "registry.ci.openshift.org/ocp-ppc64le/release-ppc64le",
     "aarch64": "registry.ci.openshift.org/ocp-arm64/release-arm64",
 }
+# The client name mapping in nightly with name on mirror
+MIRROR_CLIENTS = {
```

Comment:
This doesn't see to be a constant that needs to be used in other modules. I think it is better to be moved to `promote.py`.

Reply:
we will migrate client_sync job later, I think there will be other types of clients added here